src/components/User/newUserPopup/NewUserForm/index.js

```diff
@@ -55,10 +56,21 @@ const NewUserForm = ({
     () => [errorMessage === PASS_EQUAL_ERROR_MESSAGE ? styles.visible : styles.hidden],
     [isError],
   );
+  const editFormStyle = useMemo(
+    () => [editForm ? styles.notDisplay : styles.display],
+    [editForm],
+  );
+  const addFormStyle = useMemo(
+    () => [editForm ? styles.display : styles.notDisplay],
+    [editForm],
+  );
   const alertClasses = useMemo(() => [isError ? styles.visible : styles.hidden], [isError]);
   return (
     <div className={styles.maxWidth}>
-      <Caption classes={styles.newUserCaption}> Add new user </Caption>
+      <Caption classes={styles.newUserCaption}>
+        <p className={editFormStyle}> Add new user </p>
+        <p className={addFormStyle}> Edit user </p>
+      </Caption>
```

Comment:
Can't we just use conditional rendering?

<p>{editForm ? 'Edit user' : 'Add new user'}</p>